src/frontend/Player/createPlyrPlayer.spec.ts

```diff
@@ -7,16 +7,70 @@ jest.mock('plyr', () => {
 });
 jest.mock('jwt-decode', () => {
   return jest.fn().mockImplementation(() => ({
+    locale: 'en',
     session_id: 'abcd',
   }));
 });
+jest.mock('react-intl', () => ({
+  IntlProvider: jest.fn().mockImplementation(() => ({
+    getChildContext: jest.fn().mockReturnValue({
+      intl: {
+        formatMessage: jest.fn(),
+      },
+    }),
+  })),
+  defineMessages: jest.fn().mockReturnValue({
```

Comment:
You could probably use `mockImplementation` to just process the messages you're passed and avoid listing all the strings here

Reply:
Good idea !